### DISCUSSION TITLE
How can I setup vscode for develeopment of this repo i.e. `avr-hal`?

### DISCUSSION THREAD

**Discussion by RagibHasin** [Rahix/avr-hal#525] [Q&A]
I have set `"rust-analyzer.cargo.features": ["atmega8"]`, but it still gives me more than 100 errors like this:

```
the name `pac` is defined multiple times
`pac` must be defined only once in the type namespace of this module
```

How do you develop this repo locally?

**Comment by Rahix** [comment]
Huh, I have no trouble using `rust-analyzer` with this codebase in neovim.  Although I don't set any `"rust-analyzer.cargo.features": ["atmega8"]` like you mentioned.

I wonder whether your `r-a` config accidentally has multiple different MCU features activated at the same time?  That's the only thing that would explain multiple `pac` items...
